conda/meta.yaml

```diff
@@ -32,9 +35,10 @@ test:
     - tests
     - schema
     - setup.cfg
+    - pyproject.toml
     - .git
-  commands:
-    - pytest
+#  commands:
+#    - pytest
```

Comment:
Please remove these comment-out lines, if practical. If you want to keep them then please add a comment explaining why they are commented-out and when they can be uncommented (or discarded), preferably with a JIRA ticket #.

Reply:
Thanks for catching that. Those should not be uncommented. I just did it to be able to run the build quicker. I will uncomment them again.